dlr/dlr-engine/src/test/resources/mlrepo/model/cv/image_classification/ai/djl/dlr/resnet/metadata.json

```diff
@@ -0,0 +1,59 @@
+{
+  "metadataVersion": "0.2",
+  "resourceType": "model",
+  "application": "cv/image_classification",
+  "groupId": "ai.djl.dlr",
+  "artifactId": "resnet",
+  "name": "Resnet",
+  "description": "Resnet image classification",
+  "website": "https://github.com/neo-ai/neo-ai-dlr",
+  "licenses": {
+    "license": {
+      "name": "The Apache License, Version 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0"
+    }
+  },
+  "artifacts": [
+    {
+      "version": "0.0.1",
+      "snapshot": false,
+      "name": "resnet50",
```

Comment:
use "resnet50_v1" to avoid optModelName()

Reply:
done